text/0000-RouteInfo-Metadata.md

```diff
@@ -0,0 +1,210 @@
+- Start Date: 2018-11-02
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# RouteInfo MetaData
+
+## Summary
+
+The RFC introduces the ability to associate application specific metadata with it's corresponding `RouteInfo` object. This also adds a `metadata` field to the `RouteInfo` which will be the return value of `buildRouteInfoMetaData` for it's corresponding `Route`.
+
+```js
+// app/route/profile.js
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+
+  buildRouteInfoMetaData() {
+    return {
+      trackingKey: 'page_profile',
+      profile: {
+        viewing: this.userId,
```

Comment:
I'd _vaguely_ prefer to avoid suggesting folks should store state on the route instance. Maybe this (and the line below) should be using a user service ala `this.user.userId` and `this.user.userLocale`.